src/main/resources/db/changelog/changes/20211116000002-create-schema.yaml

```diff
@@ -1,140 +1,170 @@
                     nullable: false
+                    primaryKeyName: PK_ENT_BUN_COM_JOBS_E7
               - column:
                   name: parent_entando_bundle_job_id
-                  type: uuid
+                  type: VARCHAR(36)
                   constraints:
                     nullable: false
-                    foreignKeyName: fk_de_comp_job
+                    foreignKeyName: FK_DE_COMP_JOB_E7
                     references: entando_bundle_jobs(id)
               - column:
-                  name: name
-                  type: varchar(255)
+                  name: component_id
+                  type: VARCHAR(255)
                   constraints:
                     nullable: false
-              - column:
-                  name: error_message
-                  type: TEXT
-                  constraints:
-                    nullable: true
               - column:
                   name: checksum
-                  type: varchar(64)
+                  type: VARCHAR(64)
                   constraints:
                     nullable: false
               - column:
                   name: status
-                  type: TEXT
+                  type: VARCHAR(35)
                   constraints:
                     nullable: false
               - column:
                   name: component_type
-                  type: TEXT
+                  type: VARCHAR(35)
                   constraints:
                     nullable: false
+              - column:
+                  name: started_at
+                  type: TIMESTAMP
+              - column:
+                  name: finished_at
+                  type: TIMESTAMP
+              - column:
+                  name: action
+                  type: VARCHAR(16)
+              - column:
+                  name: install_error_code
+                  type: INT
+                  defaultValue: null
+              - column:
+                  name: install_error_message
+                  type: TEXT
+                  defaultValue: null
+              - column:
+                  name: rollback_error_code
+                  type: INT
+                  defaultValue: null
+              - column:
+                  name: rollback_error_message
+                  type: TEXT
+                  defaultValue: null
 
+        # installed_entando_bundles
         - createTable:
             tableName: installed_entando_bundles
             columns:
               - column:
                   constraints:
                     nullable: false
                     primaryKey: true
-                    primaryKeyName: installed_entando_bundles_pkey
+                    primaryKeyName: PK_INS_ENT_BUN_E7
                   name: id
                   type: VARCHAR(255)
               - column:
                   name: description
                   type: VARCHAR(255)
               - column:
                   name: image
-                  type: BLOB
+                  type: LONGVARCHAR
```

Comment:
@firegloves please make the size explicit.
Although unlikely some engine/adapter have different defaults and potentially convert it into blob.

Reply:
@w-caffiero-entando made explicit to LONGVARCHAR(30000), however this is what I obtain:

- Oracle = clob
- Mysql = longtext (no length)
- Postgres = text (no length)
- H2 = varcar (no length)

In any case, everything is working